pkg/cmd/server/bootstrappolicy/namespace_policy.go

```diff
@@ -0,0 +1,81 @@
+package bootstrappolicy
+
+import (
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/pkg/apis/rbac"
+
+	"github.com/golang/glog"
+)
+
+func addNamespaceRole(namespaceRoles map[string][]rbac.Role, namespace string, role rbac.Role) {
```

Comment:
Not seeing a use-case where these shouldn't be auto-reconciled.  Add the annotation here.